parts/kubernetes/agentpool/kubelet.service

```diff
@@ -0,0 +1,53 @@
+[Unit]
```

Comment:
There are all kinds of kubelet flags below that we're going to have to flip in multiple places. This is a recipe for skew and errors. I still don't see a concrete example where we can't have a shared `kubelet.service` file.  Happy to talk this through in person if that's easier.